packages/sdk/src/core/errors.ts

```diff
@@ -70,12 +70,19 @@ export enum ErrorCode {
    * Identity type error.
    */
   IdentityTypeNotSupported = "IdentityTypeNotSupported",
+
+  /**
+   * Authentication info error.
+   */
+  AuthorizationInfoError = "AuthorizationInfoError",
```

Comment:
Can we make the error code more clear? It may be hard to understand what does authorization info error mean.

Reply:
Fixed to AuthorizationInfoAlreadyExists error type.